lenovo/thinkpad/x260/default.nix

```diff
@@ -0,0 +1,12 @@
+{
+  imports = [
+    ../.
+    ../acpi_call.nix
+    ../../../common/cpu/intel
+  ];
+
+  # https://wiki.archlinux.org/index.php/TLP#Btrfs
+  services.tlp.extraConfig = ''
+    SATA_LINKPWR_ON_BAT=max_performance
```

Comment:
There is a discussion on this: https://wiki.archlinux.org/index.php/Talk:TLP#Btrfs
Should we make only enable this on kernel lower then 4.15?

Reply:
Was this reproducible with any other file system then BTRFS?

Reply:
@Mic92 I think we can set it to `med_power_with_dipm` on all laptops starting with 4.15 kernels, just in case.